README.md

```diff
@@ -15,6 +15,8 @@
 - [akivajgordon/tikkun.io](https://github.com/akivajgordon/tikkun.io) - The online tikkun you always wanted, but never had
 - [OpenSiddur/OpenSiddur](https://github.com/opensiddur/opensiddur) - A free tool for making custom siddurim, haggadot, and more.
 - [Yiddishe-Kop/Shas-Vilna](https://github.com/Yiddishe-Kop/Shas-Vilna) - An attempt to recreate the Vilna edition of the Talmud Bavli on the web
+- [meirlamdan/shnayim-mikra](https://github.com/meirlamdan/shnayim-mikra) - שניים מקרא ואחד תרגום - Responsive reading from any device
+- [david-lev/hebrewbooksbot](https://github.com/david-lev/hebrewbooksbot) - A bot that allows you to search and browse books from hebrewbooks.org
```

Comment:
I feel uncomfortable listing this one. I actually tried to make something similar once. Then I learned that [the HebrewBooks.org Terms of Use](https://hebrewbooks.org/media/terms.html) (section 2) say not to use bots or scrapers to access the site.

I opened https://github.com/david-lev/hebrewbooksbot/issues/3 to get more info, but I would rather omit this for now.

```suggestion
```